### PR TITLE
release: v0.12.0rc

### PR DESCRIPTION
**Pull request by iGormilhit**
## Why are you opening this PR?

For the `v0.12.0` release

## How to test?

- Check the changes files.
- Have a look to the RELEASE-NOTES.rst file.

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

File: RELEASE-NOTES.rst [hunk 1]
```diff
@@ -18,6 +18,271 @@
...
+-  Adds ``marcxml`` support to the document API, thus allowing
+   authenticated user to post ``marcxml`` records using an external
+   script. The record is added to the database with the ``_draft`` flag
+   set to true, to disable the validation of the data and to avoid the
+   record to be found in the catalog.
+
+Permission
+~~~~~~~~~~
+
+-  Adds the ``document_importer`` role for user that post records
```
Comment (mmo) [inline RELEASE-NOTES.rst:206]:
```suggestion
-  Adds the ``document_importer`` role to users posting records
```

**Review by mmo — COMMENTED** [review]
Uff!

File: CHANGES.rst [hunk 2]
```diff
@@ -17,6 +17,152 @@
...
+-  The help menu should point to the public help page if clicked from
+   public interface `#1127`_
+-  scripts: install ng-core and ui in main project `#1142`_
+   (`blankoworld`_)
+-  cypress: enhance Cypress commands precision `#1136`_ (`blankoworld`_)
+-  tests: enhance Cypress with fixtures `#1125`_ (`blankoworld`_)
+
+**Fixed bugs:**
+
+-  message ’prêt impossible : l’exemplaire est demandé par un autre
```
Comment (BadrAly) [inline CHANGES.rst:36]: missing ` at the end of "’prêt impossible"

**Review by BadrAly — APPROVED** [review]

File: CHANGES.rst (hunk 2 above)
Comment (iGormilhit) [reply CHANGES.rst:36]: @BadrAly This has been automatically fetched from the rero/rero-ils#1160 title. :smiley:

I'm going to change the issue title and regenerate the changelog.

**Pull request merged**